src/main/java/com/ibm/cloud/ibm_key_protect_api/v2/model/AllowedIPProperties.java

```diff
@@ -1,122 +0,0 @@
-/*
```

Comment:
This appears to have been replaced by the class`InstancePolicyAllowedIPPolicyDataAttributes` in where it appeared in the master branch.